### ISSUE TITLE
Add the ability to 'badge' the icon

### ISSUE THREAD

**Issue by johnofcamas** [johnofcamas/streamdeck_template#2]
Thinking here is that the icon (named icon.png) can be badged using the PIL library (I've already got this working in another plugin).

The pub_server would send a payload 
```json
{
   "action" : "badge_icon", 
   "payload" : 
   {
      "color" : "<RGB>", 
      "number" : 5
   }
}
```

We should also have a message to clear the badge

```json
{
   "action" : "reset_icon"
}
```

The `reset_icon` message would simply clear the badge

**Comment by johnofcamas** [comment]
this is moving along nicely, the only change will be the badge_icon payload.. that is going to be:
```json
{
   "action" : "badge_icon", 
   "payload" : 
   {
      "color" : [r,g,b], 
      "number" : 5
   }
}
```
where r,g,b are values from 0-255

**Comment by johnofcamas** [comment]
Resolved with bd5d072
